cli/command/image/tree.go

```diff
@@ -205,10 +206,11 @@ func printImageTree(dockerCLI command.Cli, view treeView) error {
 	untaggedColor := out.Color(tui.ColorTertiary)
 	isTerm := out.IsTerminal()
 
-	out.PrintlnWithColor(tui.ColorWarning, "WARNING: This is an experimental feature. The output may change and shouldn't be depended on.")
+	if stdoutRedirected(dockerCLI) {
+		fmt.Fprintln(dockerCLI.Err(), "WARNING: This output is designed for human readability. For machine-readable output, please use --format.")
```

Comment:
Can you suppress the err handling here?

```suggestion
		_, _ = fmt.Fprintln(dockerCLI.Err(), "WARNING: This output is designed for human readability. For machine-readable output, please use --format.")
```

(reminds me we need utilities for this; it's cumbersome to handle this everywhere)